src/widgets/_menus.scss

```diff
@@ -5,6 +5,16 @@
 
     separator {
         min-height: rem(6px);
+
+        &.horizontal {
+            border-top: 1px solid rgba(black, 0.15);
+            margin-bottom: rem(-1px);
```

Comment:
If you're not using `rem` for the border, we probably don't want to use `rem` here

Reply:
Oh, missed a place, wanted to use rem everywhere here. oops.